ui/src/main/resources/FileManagerCode/Pagination.xml

```diff
@@ -56,6 +56,16 @@
       &lt;/span&gt;
     &lt;/span&gt;
 
+    &lt;span ng-if="showResultsRange &amp;&amp; totalCount &gt; 0" class='resultsNo xwiki-livetable-pagesize'&gt;
+      &lt;span&gt;$services.localization.render('platform.livetable.pagesizeLabel')&lt;/span&gt;
```

Comment:
```suggestion
      &lt;span&gt;$escapetool.xml($services.localization.render('platform.livetable.pagesizeLabel'))&lt;/span&gt;
```

Reply:
Should I add escapes for other similar `$services.localisation.render` calls which were not added as part of this commit?